tests/formats/test_rdf.py

```diff
@@ -0,0 +1,149 @@
+from _pytest.fixtures import FixtureRequest
+
+from spinta.core.config import RawConfig
+from spinta.testing.client import create_test_client
+from spinta.testing.data import pushdata
+from spinta.testing.manifest import bootstrap_manifest
+
+
+def test_rdf_get_all_without_namespaces(
+    rc: RawConfig,
+    postgresql: str,
+    request: FixtureRequest,
+):
+    context = bootstrap_manifest(rc, '''
+    d | r | b | m | property | type   | ref     | access  | uri
+    example/rdf              |        |         |         |
+      |   |   | Country      |        | name    |         | dcat:country
+      |   |   |   | name     | string |         | open    | dct:name
+      |   |   | City         |        | name    |         | dcat:city
+      |   |   |   | name     | string |         | open    | dct:name
+      |   |   |   | country  | ref    | Country | open    | dct:country
+    ''', backend=postgresql, request=request)
+    app = create_test_client(context)
+    app.authmodel('example/rdf', ['insert', 'getall'])
+
+    country = pushdata(app, '/example/rdf/Country', {'name': 'Lithuania'})
+    city1 = pushdata(app, '/example/rdf/City', {
+        'name': 'Vilnius',
+        'country': {'_id': country['_id']},
+    })
+    city2 = pushdata(app, '/example/rdf/City', {
+        'name': 'Kaunas',
+        'country': {'_id': country['_id']},
+    })
+
+    res = app.get("/example/rdf/City/:format/rdf").text
+    assert res == f'<Rdf>\n'\
+                  f'  <City>\n'\
```

Comment:
If `uri` is given, use it here, in this case it should be `dcat:city`, if `uri` is not given, a `<Description rdf:about="example/rdf/City">` form should be used.